docs/Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+1.12.4 (?? Oct 2023)
+* [#4992](https://github.com/TouK/nussknacker/pull/4992) Fix: Sort PeriodicProcessDeployments by `runAt` and `createdAt` and instead of `runAt` only.
```

Comment:
please edit it taking into consideration the users' perspective

Reply:
Thanks, changed a little bit